clang/lib/AST/ASTContext.cpp

```diff
@@ -4339,7 +4339,6 @@ ASTContext::getBuiltinVectorTypeInfo(const BuiltinType *Ty) const {
   switch (Ty->getKind()) {
   default:
     llvm_unreachable("Unsupported builtin vector type");
-
```

Comment:
Please can we keep these blank lines.  With all the continuation markers they help partition the related defines and includes.